tonto/routers_tonto.py

```diff
@@ -0,0 +1,37 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import cowboys
+from tonto import crud, schemas
+from settings import bd_coneccion
+
+router = APIRouter()
+
+
+@router.get ("/tonto", response_model = cowboys.schemas.CrearCowboyRespuesta)
+def seleccionar_tonto (db: Session = Depends (bd_coneccion.get_db)):
```

Comment:
```suggestion
@router.get ("/tontos", response_model = cowboys.schemas.CrearCowboyRespuesta)
def obtener_lista_tontos (db: Session = Depends (bd_coneccion.get_db)):
```

Este endpoint se tiene ser `/tontos` y tiene que devolver la lista de todos los tontos con sus titulos para ser compatible con la pagina actual.